.travis.yml

```diff
@@ -3,10 +3,15 @@ language: php
 php:
     - 7.3
 
+cache:
+    directories:
+        - vendor
```

Comment:
Do not cache the vendor folder, vendor may change, and because you cached `$HOME/.composer/cache` you, composer will be fast enough "downloading" (meaning copying) files from that cache.